app/workers/scheduled_publishing_job.rb

```diff
@@ -16,21 +15,19 @@ def perform(id)
       PublishService.new(edition.document)
                     .publish(user: user, with_review: reviewed)
     end
-  rescue ActiveRecord::RecordNotFound
```

Comment:
Just wondering why we're no longer raise a specific error here?

Reply:
Good question :-). Previously we were rescuing this error in order to abort immediately, since it's impossible to send an email for an edition that does not exist e.g. if it's been wiped on integration. Because we're using ActiveJob, which aborts by default (vs. Sidekiq), we can just let the exception bubble-up, rather than having to instruct Sidekiq to abort.

Reply:
Ok that makes sense.  I wonder if it would still be helpful from a debugging perspective to log that the edition was not found in a scheduled publishing context?

Reply:
It does now :-).